gulpfile.js

```diff
@@ -40,10 +41,15 @@ let zaExtra7z = path.join(zaRoot, '7z920_extra.7z');
 let rhZip = path.join(zaRoot, 'resource_hacker.zip');
 let rhExe = path.join(zaRoot, 'ResourceHacker.exe');
 let zaElectronPackage = path.join(zaRoot, artifactName + '-win32-x64');
-let bundled7z = path.join(zaRoot, artifactName +'-win32-x64.7z');
-let installerExe = resolveInstallerExePath('');
-
-gulp.task('transpile:app', function() {
+let bundled7z = path.join(zaRoot, artifactName +'-win32-x64.7z');
+let installerExe = resolveInstallerExePath('');
+
+process.on('uncaughtException', function(err) {

```

Comment:
In our case to avoid problems like missing callbacks I would even re-throw the error, just to see the stacktrace.
```  JavaScript
process.on('uncaughtException', function(err) {
    if(err) {
      throw err;
    }
});
```
then I would see stacktrace for my missing callback 
```
C:\p\rh\dpi\gulpfile.js:50
      throw err;
      ^

TypeError: cb is not a function
    at createSHA256File (C:\p\rh\dpi\gulpfile.js:212:10)
    at WriteStream.<anonymous> (C:\p\rh\dpi\gulpfile.js:301:19)
    at emitNone (events.js:85:20)
    at WriteStream.emit (events.js:179:7)
    at finishMaybe (_stream_writable.js:478:14)
    at afterWrite (_stream_writable.js:356:3)
    at onwrite (_stream_writable.js:346:7)
    at WritableState.onwrite (_stream_writable.js:89:5)
    at fs.js:1951:5
    at FSReqWrap.wrapper [as oncomplete] (fs.js:667:5)
```